src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -4217,7 +4217,11 @@ namespace HeatBalanceAirManager {
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         // na
 
-        ManageHVAC(state, OutputFiles::getSingleton());
+        if(DataGlobals::externalHVACManager) {
```

Comment:
That is indeed a minimal set of changes.

Reply:
Oh yeah!